charts/private-action-runner/examples/config.yaml

```diff
@@ -35,3 +35,29 @@ runners:
         - com.datadoghq.kubernetes.core.listPod
         - com.datadoghq.http.request
         - com.datadoghq.jenkins.buildJenkinsJob
+
+connectionCredentials:
+  basicAuth:
+    credentials:
+      - key: "basic.json"
```

Comment:
suggestion - replace `key` with `fileName`. it's more descriptive and is also consistent with the connections interface where you need a file name